src/wrapped/wrappedpulse.c

```diff
@@ -891,11 +891,11 @@ static void* find_time_restart_Fct(void* fct)
     return NULL;
 }
 // time_free
-#define GO(A)                                       \
-static uintptr_t my_time_free_fct_##A = 0;          \
-static void* my_time_free_##A(void* e)              \
-{                                                   \
-    RunFunctionFmt(my_time_free_fct_##A, "p", e);   \
+#define GO(A)                                                   \
```

Comment:
it's a `void` callback https://freedesktop.org/software/pulseaudio/doxygen/structpa__mainloop__api.html

Reply:
Fixed.